.gitignore

```diff
@@ -60,5 +60,5 @@ build/
 
 ## Local env file having environment variable definitions for docker-compose.yml and other files
 ## We should not commit this to prevent leaking confidential information
-.env*
+.env
```

Comment:
@ilkerCelimli Can we revert the change here? There shouldn't have been a need for such a modification specific to the development. Especially the changes made in the `application.yml` were preventing the application from being launched, so I had to remove them.